plugins/chain_plugin/chain_plugin.cpp

```diff
@@ -1930,6 +1930,47 @@ chain::signed_block_ptr read_only::get_block(const read_only::get_block_params&
    return block;
 }
 
+read_only::get_block_header_result read_only::get_block_header(const read_only::get_block_header_params& params, const fc::time_point& deadline) const{
+   std::optional<uint64_t> block_num;
+
+   EOS_ASSERT( !params.block_num_or_id.empty() && params.block_num_or_id.size() <= 64,
+               chain::block_id_type_exception,
+               "Invalid Block number or ID, must be greater than 0 and less than 64 characters"
```

Comment:
You have an extra equal to 64 checked while greater than 0 is not checked. This is not blocking. You can fix in RC.

Reply:
I believe the assertion condition is right but the message description was problematic, fixed in commit af1dcf23cf99a4236ef589f87e8c057167e0fece